### DISCUSSION TITLE
Export single bitmap image as 1-layer slicer file?

### DISCUSSION THREAD

**Discussion by rspeir** [sn4k3/UVtools#238] [Q&A] [answered]
I'm experimenting with using my Elegoo Mars2 to expose UV-sensitized copper clad board to make circuit boards, and as such, I need a way to get the printer to display a single image for several seconds to expose the board.

I see that I can open .png, .gif, .jpg, etc image files in UVtools, but it appears that I can't simply export an opened image to a slicer file. 

I realize this is probably doable through scripting, but I'm getting kind of lost in figuring out which methods and data structures in the UVtools.Core codebase I would need to use. Could someone please provide a little bit of guidance? 

Thank you!

**Comment by sn4k3** [comment, marked as answer]
You need to use the Actions -> Import under any sliced file for your printer and modify it with the image you want to expose.

Import your pcb image as:
![image](https://user-images.githubusercontent.com/113281/125016838-eac68a00-e069-11eb-966a-3c9ace2fe29d.png)

Result:
![image](https://user-images.githubusercontent.com/113281/125017190-8c4ddb80-e06a-11eb-91ee-857d10507b6e.png)

If required is also possible to invert:
![image](https://user-images.githubusercontent.com/113281/125017320-cae39600-e06a-11eb-8034-9f2471d52fa4.png)


Also here: https://github.com/sn4k3/UVtools/issues/148

**Reply by rspeir** [reply]
Wow, that was fast! Thank you so much!

**Reply by sn4k3** [reply]
> Wow, that was fast! Thank you so much!

Fast and easy ;) No scripting needed. In fact it can do so much that most people don't know much of the functions.
